.github/workflows/tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Test
+
+on:
+  pull_request:
+    branches:
+      - development
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10"]
```

Comment:
I think we can take Python 3.9 out of the matrix. We don't actually run it on 3.9 anywhere.